packages/ui/src/utils/format-safe-identifier.ts

```diff
@@ -0,0 +1,15 @@
+import { stringToFormattedPhoneString } from '~/common/phone-number-field/utils';
```

Comment:
Nit: I think we should move this to `~/utils` if it's something we plan to use across different areas of the codebase; this'll avoid the likelihood of circular imports (and just a little tidier)

I may go ahead and just make this change to avoid the back-and-forth

Reply:
Ooof actually just seen how this also relies on `data` etc. too. Let's come back to this